src/index.ts

```diff
@@ -195,7 +195,16 @@ type Middleware = (request: Request, response: Response) => Promise<void>;
  * Middleware for express; takes an options object or function as input to
  * configure behavior, and returns an express middleware.
  */
-export function graphqlHTTP(options: Options): Middleware {
+export function graphqlHTTP(
+  options: Options,
+  /**
+   * An optional function which will get params from request instead of
+   * default getGraphQLParams
+   */
+  customGraphQLParamsFn?: (
```

Comment:
@TrySound is there a reason you didn't want to add this to the `Options` type?

Reply:
I cannot because of options getter which depends on params. If we could get rid of it Im good to move into options

> It would look better in options though options getter depends on parsed
options.